core/src/main/kotlin/riven/core/service/catalog/ManifestResolverService.kt

```
@@ -103,10 +107,65 @@ class ManifestResolverService(
+     * declared attributes. Returns null (with a WARN log) on any cross-validation failure
+     * or malformed payload — consistent with relationship/field-mapping handling.
+     * Structural rejection happens earlier at JSON-schema validation time.
+     */
+    private fun parseConnotationSignals(
+        node: JsonNode,
+        knownAttributeKeys: Set<String>,
+        entityTypeKey: String?,
+    ): ConnotationSignals? {
+        val signals = try {
+            objectMapper.treeToValue(node, ConnotationSignals::class.java)
+        } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is too generic. Prefer catching specific exceptions to the case that is currently handled.

(detekt.exceptions.TooGenericExceptionCaught)